docs/interpreter/java.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: Java interpreter in Apache Zeppelin
+description: Run Java code and any distributed java computation library by importing the dependencies in the interpreter configuration.
+group: interpreter
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+{% include JB/setup %}
+
+# Java interpreter for Apache Zeppelin
+
+<div id="toc"></div>
+
+## How to use
+Basically, you can write normal java code. You should write the main method inside a class because the interpreter invoke this main to execute the code. Unlike Zeppelin normal pattern, each paragraph is considered as a separate job, there isn't any relation to any other paragraph.
```

Comment:
perhaps clarify " each paragraph is considered as a separate job, there isn't any relation to any other paragraph."
does it mean I can't share variables in different paragraphs? I think it will greatly help if this is documented

Reply:
At the moment there isn't a way to share variable unless you use a clustered environment.
For example, when I will implement the Hazelcast interpreter, one could store a variable in the hazelcast cluster as part of one job and retrieve the same variable as part of another job.

As in previous comments with @zjffdu when we will move to JShell it will be easier to introduce the support for ZeppelinContext which will allow also to share variables.